toontown/spellbook/MagicWordIndex.py

```diff
@@ -316,7 +316,20 @@ def handleWord(self, invoker, avId, toon, *args):
         toon.b_setTickets(RaceGlobals.MaxTickets)
         maxTrophies = RaceGlobals.NumTrophies + RaceGlobals.NumCups
         toon.b_setKartingTrophies(range(1, maxTrophies + 1))
-        toon.b_setTickets(99999)
```

Comment:
Why is this removed ?

Reply:
It is not necessary to have it twice. The max tickets are already set three lines above it on line 316:

```toon.b_setTickets(RaceGlobals.MaxTickets)```